rotkehlchen/tests/db/test_db_upgrades.py

```diff
@@ -2620,6 +2622,31 @@ def test_upgrade_db_45_to_46(user_data_dir: 'Path', messages_aggregator):
         icon_path = icons_dir / f'{urllib.parse.quote_plus(A_COW.identifier)}_small.png'
         icon_path.write_bytes(b'0x0x')
 
+        assert cursor.execute(
+            "SELECT COUNT(*) FROM pragma_table_info('history_events') WHERE name='extra_data'",
+        ).fetchone()[0] == 0
+        assert cursor.execute(
+            "SELECT COUNT(*) FROM pragma_table_info('evm_events_info') WHERE name='extra_data'",
+        ).fetchone()[0] == 1
+        events_with_extra_data = [row[0] for row in cursor.execute(
+            "SELECT identifier FROM evm_events_info WHERE extra_data != '';",
+        )]
+        existing_evm_event = cursor.execute('SELECT * FROM history_events WHERE identifier = "35"').fetchone()  # noqa: E501
+        existing_evm_event_extra_data = cursor.execute('SELECT extra_data FROM evm_events_info WHERE identifier = "35"').fetchone()[0]  # noqa: E501
+        assert existing_evm_event_extra_data == '{"airdrop_identifier": "elfi"}'
+
+    # Add a plain history event to the db to be checked after upgrade that it wasn't modified
+    # Note that it has to be manually inserted here since the functions for creating
+    # history events now expect there to be an extra_data column in history_events
+    history_event_bindings = [HistoryBaseEntryType.HISTORY_EVENT.value, 'TEST1', 0, 1, Location.KRAKEN.serialize_for_db(), 'Somewhere', A_ETH.identifier, 1, 3000, 'Just a test event', HistoryEventType.INFORMATIONAL.value, HistoryEventSubType.NONE.value]  # noqa: E501
+    with db_v5.conn.write_ctx() as write_cursor:
+        write_cursor.execute(
```

Comment:
nitpick: not related to this PR but the v5 I believe is a typo. Should be 45

Reply:
+1